app/views/shared/_comment.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if (post.comments.empty?) %>
+  <p>No comment yet.</p>
+<% else %>
+  <% post.comments.each do |comment| %>
+    <ul>
```

Comment:
- [ ] It would be nice if you use your `most_recent_commnets` method here to get 5 most recent comments as requested in the requirements :+1: